libs/dh/metering-point/feature-overview/src/lib/dh-can-see.directive.spec.ts

```diff
@@ -0,0 +1,136 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+/* eslint-disable sonarjs/no-duplicate-string */
+import { render } from '@testing-library/angular';
+
+import {
+  EicFunction,
+  ElectricityMarketMeteringPointType,
+} from '@energinet-datahub/dh/shared/domain/graphql';
+import { DhActorStorage } from '@energinet-datahub/dh/shared/feature-authorization';
+
+import { DhCanSeeDirective, PropertyName } from './dh-can-see.directive';
+import { MeteringPointDetails } from './types';
+
+function getPropertyName(propertyName: PropertyName) {
+  return propertyName;
+}
+
+type Params = {
+  canSeeProperty: PropertyName;
+  mpType: ElectricityMarketMeteringPointType;
+};
+
+describe(DhCanSeeDirective, () => {
```

Comment:
Praise: nice with tests here

Reply:
It's not much but at least a start. More tests can be added if we find a bug where something that shouldn't be shown is displayed. We have a starting point now.